server/paths/apply.ts

```diff
@@ -15,6 +15,7 @@ const paths = {
     index: applicationsPath,
     new: applicationsPath.path('new'),
     beforeYouStart: applicationsPath.path('before-you-start'),
+    applicationOrigin: applicationsPath.path('application-origin'),
```

Comment:
do we know if application origin is a term used by referrers, or just something we have come up with? If so maybe worth validating that and whether we should use an alternative in user facing text, e.g. application type?

Reply:
Good spot on the URL, I tried to keep `type` in the user facing text, whilst not causing confusion with the other `ApplicationType` used to differentiate between the CAS services. I think we should change the path to 'application-type' in this case 👍🏾